src/redteam/index.ts

```diff
@@ -155,14 +157,19 @@ const Strategies: Strategy[] = [
   },
 ];
 
-function validatePlugins(plugins: { id: string; numTests: number }[]): void {
+function validatePlugins(
+  plugins: { id: string; numTests: number; config?: Record<string, any> }[],
+): void {
   const invalidPlugins = plugins.filter((plugin) => !Plugins.map((p) => p.key).includes(plugin.id));
   if (invalidPlugins.length > 0) {
     const validPluginsString = Plugins.map((p) => p.key).join(', ');
     const invalidPluginsString = invalidPlugins.map((p) => p.id).join(', ');
-    throw new Error(
-      `Invalid plugin(s): ${invalidPluginsString}. Valid plugins are: ${validPluginsString}`,
+    logger.error(
+      dedent`Invalid plugin(s): ${invalidPluginsString}. 
+      
+      ${chalk.white(`Valid plugins are: ${validPluginsString}`)}`,
```

Comment:
white will be invisible on light mode terminals (yes those exist)
```suggestion
      ${chalk.green(`Valid plugins are: ${validPluginsString}`)}`,
```